crowbar_framework/app/controllers/api/backups_controller.rb

```diff
@@ -35,6 +59,17 @@ def show
   param :backup, Hash, desc: "Backup info", required: true do
     param :name, String, desc: "Name of the backup", required: true
   end
+  example '
+  {
+    "id": 1,
+    "name": "testbackup",
+    "version": "4.0",
+    "size": 76815,
+    "created_at": "2016-09-27T06:05:10.208Z",
+    "updated_at": "2016-09-27T06:05:10.208Z",
+    "migration_level": 20160819142156
+  }
```

Comment:
Well, I guess we can use the id to download the backup file. Although I would expect an href > download > link attribute to get it from this response